client/src/app/pages/applications/applications-table/applications-table.tsx

```diff
@@ -499,7 +503,38 @@ export const ApplicationsTable: React.FC = () => {
         ],
         getItemValue: (item) => normalizeRisk(item.risk) ?? "",
       },
+
+      {
+        categoryKey: "analysis",
+        title: t("terms.analysis"),
+        type: FilterType.multiselect,
+        placeholderText:
+          t("actions.filterBy", {
+            what: t("terms.analysis").toLowerCase(),
+          }) + "...",
+
+        selectOptions: Object.values(applications)
+          .map((a) => {
+            let value = a?.tasks.currentAnalyzer?.state || "No Task";
+
+            if (value === "No Task") {
+              value = "No task";
+            }
+
+            let label = taskStateToAnalyze.get(value as TaskState) || value;
+
+            if (label === "NotStarted") {
+              label = "Not started";
+            }
+            return { value, label };
+          })
+          .filter((v, i, a) => a.findIndex((v2) => v2.label === v.label) === i)
+          .sort((a, b) => a.value.localeCompare(b.value)),
```

Comment:
we should sort on label - note also that `universalComparator` helper adds some extra power to `localeCompare` ("natural sorting")